test/export/test_draft_export.py

```diff
@@ -4,8 +4,11 @@
 import unittest
 
 import torch
+from torch._library.fake_profile import MissingOpProfile, OpProfile, TensorMetadata
```

Comment:
Can you provide some more context in the PR body for what this PR is doing?

Reply:
updated!